iothub/device/src/Microsoft.Azure.Devices.Client.csproj

```diff
@@ -126,4 +120,28 @@
     <PackageReference Include="Microsoft.AspNet.WebApi.Client" Version="5.2.3" />
     <PackageReference Include="Microsoft.Owin" Version="4.0.0" />
   </ItemGroup>
+
```

Comment:
Pretty sure the plan is to only target netstandard2.0 in v2. If it makes your life easier, I'd suggest removing all other targets.